backend/app/controller.py

```
@@ -30,4 +34,45 @@ def health_check():
+            if obj['confidence'] >= confidence_threshold
+        ]
+        
+        return jsonify({
+            'success': True,
+            'objects': filtered_objects,
+            'count': len(filtered_objects),
+            'mode': 'simulation',
+            'message': 'Frontend uses TensorFlow.js for real-time detection'
+        })
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)